src/Form/Type/Operator/ContainsOperatorType.php

```diff
@@ -31,6 +33,8 @@ public function configureOptions(OptionsResolver $resolver)
                 'label_type_contains' => self::TYPE_CONTAINS,
                 'label_type_not_contains' => self::TYPE_NOT_CONTAINS,
                 'label_type_equals' => self::TYPE_EQUAL,
+                'label_type_starts_with' => self::TYPE_STARTS_WITH,
+                'label_type_ends_with' => self::TYPE_ENDS_WITH,
```

Comment:
This is a BC Break, everybody using this operator will have two new options that they possible don't use or have non-sense in their situation.

For example, this is used by our `ChoiceType`.

It seems better to create a new Operator.

Reply:
Thanks for review. Can't think of a name. You got a name for it?

Reply:
What about StringOperatorType ?

There is already a NumberOperatorType, DateOperatorType, ...

Reply:
Done.